packaging/installer/kickstart-static64.sh

```diff
@@ -111,51 +130,47 @@ if [ "$(uname -s)" != "Linux" ]; then
 fi
 
 # ---------------------------------------------------------------------------------------------------------------------
-
-# Check if tmp is mounted as noexec
-if grep -Eq '^[^ ]+ /tmp [^ ]+ ([^ ]*,)?noexec[, ]' /proc/mounts; then
-	pattern="$(pwd)/netdata-kickstart-static-XXXXXX"
-else
-	pattern="/tmp/netdata-kickstart-static-XXXXXX"
-fi
-
-tmpdir="$(mktemp -d $pattern)"
-cd "${tmpdir}" || :
-
-progress "Downloading static netdata binary: ${NIGHTLY_PACKAGE_TARBALL}"
-
-download "${NIGHTLY_PACKAGE_CHECKSUM}" "${tmpdir}/sha256sum.txt"
-download "${NIGHTLY_PACKAGE_TARBALL}" "${tmpdir}/netdata-latest.gz.run"
-if ! grep netdata-latest.gz.run sha256sum.txt | sha256sum --check - >/dev/null 2>&1; then
-	failed "Static binary checksum validation failed. Stopping netdata installation and leaving binary in ${tmpdir}"
-fi
-
-# ---------------------------------------------------------------------------------------------------------------------
-
 opts=
 inner_opts=
-while [ ! -z "${1}" ]; do
+RELEASE_CHANNEL="nightly"
+while [ -n "${1}" ]; do
 	if [ "${1}" = "--dont-wait" ] || [ "${1}" = "--non-interactive" ] || [ "${1}" = "--accept" ]; then
 		opts="${opts} --accept"
 	elif [ "${1}" = "--dont-start-it" ]; then
 		inner_opts="${inner_opts} ${1}"
+	elif [ "${1}" = "--stable-channel" ]; then
+		# echo >&2 "netdata will not auto-update"
+		RELEASE_CHANNEL="stable"
+		shift 1
 	else
 		echo >&2 "Unknown option '${1}'"
 		exit 1
 	fi
 	shift
 done
-[ ! -z "${inner_opts}" ] && inner_opts="-- ${inner_opts}"
+[ -n "${inner_opts}" ] && inner_opts="-- ${inner_opts}"
 
 # ---------------------------------------------------------------------------------------------------------------------
+TMPDIR=$(create_tmp_directory)
+cd "${TMPDIR}" || :
```

Comment:
Why `|| :`? Do we need to cd or not?

Reply:
I saw it was there before, but still doesn't make sense.

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2164

Reply:
It's not `|| exit` though, it's `|| :`, i.e. treat it as no failure.